ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-input-subtext/advanced-gas-fee-input-subtext.js

```diff
@@ -1,42 +1,71 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 
+import { isNullish } from '../../../../helpers/utils/util';
+import { formatGasFeeOrFeeRange } from '../../../../helpers/utils/gas';
+import { I18nContext } from '../../../../contexts/i18n';
 import Box from '../../../ui/box';
-import I18nValue from '../../../ui/i18n-value';
 import LoadingHeartBeat from '../../../ui/loading-heartbeat';
 
-const AdvancedGasFeeInputSubtext = ({ latest, historical, feeTrend }) => {
+const FEE_TRENDS = ['up', 'down', 'level'];
+
+const AdvancedGasFeeInputSubtext = ({ latest, historical, trend }) => {
+  const t = useContext(I18nContext);
   return (
-    <Box className="advanced-gas-fee-input-subtext">
-      <Box display="flex" alignItems="center">
-        <span className="advanced-gas-fee-input-subtext__label">
-          <I18nValue messageKey="currentTitle" />
-        </span>
-        <span className="advanced-gas-fee-input-subtext__value">
-          <LoadingHeartBeat />
-          {latest}
-        </span>
-        <span className={`advanced-gas-fee-input-subtext__${feeTrend}`}>
-          <img src={`./images/${feeTrend}-arrow.svg`} alt="feeTrend-arrow" />
-        </span>
-      </Box>
-      <Box>
-        <span className="advanced-gas-fee-input-subtext__label">
-          <I18nValue messageKey="twelveHrTitle" />
-        </span>
-        <span className="advanced-gas-fee-input-subtext__value">
-          <LoadingHeartBeat />
-          {historical}
-        </span>
-      </Box>
+    <Box
+      display="flex"
+      alignItems="center"
+      gap={4}
+      className="advanced-gas-fee-input-subtext"
+    >
+      {isNullish(latest) ? null : (
+        <Box display="flex" alignItems="center" data-testid="latest">
+          <span className="advanced-gas-fee-input-subtext__label">
+            {t('currentTitle')}
+          </span>
+          <span className="advanced-gas-fee-input-subtext__value">
+            <LoadingHeartBeat />
+            {formatGasFeeOrFeeRange(latest)}
+          </span>
+          {FEE_TRENDS.includes(trend) ? (
```

Comment:
Could `trend` be defined and something other than one of the values in `FEE_TRENDS`?

Reply:
Its [`proptype`](https://github.com/MetaMask/metamask-extension/pull/13865/files#diff-e17a799c978102028bdef5b55bcecf881e0bc5ee0fc0d750acbb294f13128b0bR68) suggests not. Is this just a check that is defined?

Reply:
`trend` "shouldn't" be anything other than one of the values in `FEE_TRENDS`, but it could be. This value is coming from the Gas API, so my thinking is that if the API changes such that it provides a different value, it shouldn't affect the UI and show a broken image. Does that answer your question?

Reply:
Yes that makes sense